doc/development/deprecations.rst

```diff
@@ -132,6 +132,27 @@ Pending deprecations
   - Deprecated in v0.32
   - Will be removed in v0.33
 
+* The following decorator syntax for batch transforms has been deprecated:
```

Comment:
I would say just transform, it is also deprecated for qfuncs.

Reply:
Should I add a separate deprecation message for qfuncs?